### PR TITLE
feat: add system table

### LINKED ISSUE #697: Create system table when OpenMLDB startup
We plan to store the information of jobs which are submitted  by `TaskManager` in the system table called `JOB_INFO`.  All system table will be stored in `__INTERAL_DB` database.  `__INTERAL_DB` database and `JOB_INFO` table will be create during OpenMLDB startup.

### PR DESCRIPTION
**Pull request by dl239**

### CODE REVIEW

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: release/conf/nameserver.flags [hunk 1]
```diff
@@ -34,3 +34,5 @@
 
 #--replica_num=3
 #--partition_num=8
+--system_table_replica_num=2
```
Comment (vagetablechicken) [inline release/conf/nameserver.flags:37]: Why 2?

File: src/nameserver/name_server_impl.cc [hunk 2]
```diff
@@ -5583,10 +5583,25 @@ int NameServerImpl::CreateChangeLeaderOPTask(std::shared_ptr<OPData> op_data) {
...
+            LOG(FATAL) << "tablet num " << tablets_.size()
+                << " is less then system table replica num " << FLAGS_system_table_replica_num;
+            exit(1);
+        }
+        auto status = CreateDatabase(INTERNAL_DB);
+        if (!status.OK() && status.code != ::openmldb::base::ReturnCode::kDatabaseAlreadyExists) {
+            LOG(FATAL) << "create internal database failed";
+            exit(1);
+        }
+        if (FLAGS_system_table_replica_num > 0 && !CreateSystemTable("JOB_INFO", SystemTableType::kJobInfo).OK()) {
```
Comment (vagetablechicken) [inline src/nameserver/name_server_impl.cc]: Should we define the table name "JOB_INFO" here?  or "JOB_INFO" should be a constant?

File: release/conf/nameserver.flags (hunk 1 above)
Comment (dl239) [reply release/conf/nameserver.flags:37]: gurantee high availability

Comment (vagetablechicken) [reply release/conf/nameserver.flags:37]: but this flags file is for stand-alone, we just have one tablet server, do 2 replicas make sense?

Comment (dl239) [reply release/conf/nameserver.flags:37]: This conf file is not for stand alone only and system table will not be created in standalone mode currently. If necessary, we will support in future

Comment (vagetablechicken) [reply release/conf/nameserver.flags:37]: ok

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**